travel/src/main/java/com/tx/travel/model/ExchangeRate.java

```diff
@@ -0,0 +1,29 @@
+package com.tx.travel.model;
+
+import jdk.jfr.Unsigned;
+import org.hibernate.validator.constraints.Length;
+
+import javax.persistence.*;
```

Comment:
We should avoid wildcard imports, import only necessary classes 

check: https://www.javadude.com/posts/20040522-import-on-demand-is-evil/